features/magnetometer.yml

```diff
@@ -0,0 +1,10 @@
+name: Magnetometer
+description: The magnetometer API grants access to the device magnetometer.
+spec: https://w3c.github.io/magnetometer/
+group: sensors
+compat_features:
+  - api.Magnetometer
+  - api.Magnetometer.Magnetometer
+  - api.Magnetometer.x
+  - api.Magnetometer.y
+  - api.Magnetometer.z
```

Comment:
Looking at https://developer.mozilla.org/en-US/docs/Web/API/Magnetometer#example, I am a little concerned that this is closely bound to the `Sensor` API and its events. I don't think it's a blocker, but I think it would be good to leave a TODO comment above this compat_features list saying that it overlaps and that we need to do something about it when https://github.com/web-platform-dx/web-features/issues/1173 is resolved.

Reply:
I can think of two solutions:

- Marking BCD keys that relate to base classes that developers are never expected to use as "ignored", so we never have to care about them.
- Duplicating these BCD keys across all of the features where they're used.

Reply:
```suggestion
  - api.Magnetometer
  - api.Magnetometer.Magnetometer
  - api.Magnetometer.x
  - api.Magnetometer.y
  - api.Magnetometer.z
# Enable when overlapping keys are allowed, https://github.com/web-platform-dx/web-features/issues/1173
 #  - api.Sensor
 #  - api.Sensor.activate_event
 #  - api.Sensor.activated
 #  - api.Sensor.error_event
 #  - api.Sensor.hasReading
 #  - api.Sensor.reading_event
 #  - api.Sensor.start
 #  - api.Sensor.stop
 #  - api.Sensor.timestamp
 #  - api.SensorErrorEvent
 #  - api.SensorErrorEvent.SensorErrorEvent
 #  - api.SensorErrorEvent.error
```